src/foam/u2/view/agreementView.js

```diff
@@ -0,0 +1,56 @@
+foam.CLASS({
+  package: 'foam.u2.view',
```

Comment:
Is this the best package for this? Shouldn't it be with the rest of the file code?

Reply:
Yeah, let's keep things together.  
Realize this approach has been used in the past, but let's stop doing that.